ni/measurementlink/sessionmanagement/v1/session_management_service.proto

```diff
@@ -275,4 +282,20 @@ message GetAllRegisteredMultiplexerSessionsResponse{
 message ResolvedPinsOrRelays {
   // List of pin or relay names in the pin or relay group.
   repeated string pin_or_relay_names = 1;
+}
+
+message GetSessionsRequest {
+  // Optional. Instrument type ID of the instruments.
+  string instrument_type_id = 1;
+
+  // Optional. User-defined identifier for the multiplexers.
+  string multiplexer_type_id = 2; 
```

Comment:
Add:

"If unspecified, information for all registered multiplexer types is returned."